### PR TITLE
Clarify how IPs are collected for JS SDKs

### PR DESCRIPTION
**Pull request by lforst**
Fixes https://github.com/getsentry/sentry-docs/issues/5901

### CODE REVIEW

File: src/platforms/common/enriching-events/identify-user.mdx [hunk 1]
```diff
@@ -66,7 +66,8 @@ The user segment, for apps that divide users in user segments.
 
 ### `ip_address`
 
-The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user. Sentry will attempt to pull this from the HTTP request data (`request.env.REMOTE_ADDR` field in JSON), if available. That might require <PlatformIdentifier name="send-default-pii" /> set to `true` in the SDK options.
+The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user.
+Serverside SDKs that instrument incoming requests will attempt to pull the IP address from the HTTP request data (`request.env.REMOTE_ADDR` field in JSON), if available. <PlatformSection notSupported={["javascript", "node"]}>That might require <PlatformIdentifier name="send-default-pii" /> set to `true` in the SDK options.</PlatformSection><PlatformSection supported={["javascript", "node"]}>This requires you to set the `include.ip` option to `true` in the [`RequestData`](https://docs.sentry.io/platforms/node/guides/connect/configuration/integrations/default-integrations/#requestdata) integration.</PlatformSection>
```
Comment (AbhiPrasad) [inline src/platforms/common/enriching-events/identify-user.mdx:70]: > This requires you to set the `include.ip` option to `true` in the [`RequestData`]

Doesn't this only apply to node?

Comment (AbhiPrasad) [reply src/platforms/common/enriching-events/identify-user.mdx:70]: Hmm - I'd rather have it be the correct source of truth, this split is something we should solve as an aside (cc @lizokm)

Comment (lforst) [reply src/platforms/common/enriching-events/identify-user.mdx:70]: Aight will update!

Comment (lforst) [reply src/platforms/common/enriching-events/identify-user.mdx:70]: Should I remove the entire paragraph for node though?

Comment (AbhiPrasad) [reply src/platforms/common/enriching-events/identify-user.mdx:70]: It's still technically true for node - the ip address will just be of the server then though. Let's keep it, maybe we can come back to it later.

Comment (lforst) [reply src/platforms/common/enriching-events/identify-user.mdx:70]: Sorry, edited my previous comment. I meant the paragraph is technically not relevant for browser.

Comment (AbhiPrasad) [reply src/platforms/common/enriching-events/identify-user.mdx:70]: Yes, I think that's a good idea.

**Review by AbhiPrasad — APPROVED** [review]

File: src/platforms/common/enriching-events/identify-user.mdx (hunk 1 above)
Comment (lforst) [reply src/platforms/common/enriching-events/identify-user.mdx:70]: Discussed in person: too much headache in the include-exclude logic so we'll leave it in.

**Pull request merged**